server/public/js/toggleVisibility.js

```diff
@@ -0,0 +1,10 @@
+function toggleVisibility(elementIds) {
+  elementIds.forEach(item => {
+    let x = document.getElementById(item);
+    if (x.style.display === "none") {
```

Comment:
Add and remove Gov UK classes rather than altering element styles to avoid use of inline styles:
https://design-system.service.gov.uk/styles/layout/#override-how-elements-display